contentcuration/contentcuration/frontend/shared/views/form/MultiSelect.vue

```diff
@@ -112,6 +114,15 @@
   /deep/ .v-chip__content,
   .text-truncate {
     max-width: 100%;
+    overflow: hidden;
+    white-space: nowrap;
+  }
+
+  .text-truncate-ellipsis {
+    max-width: 200px;
```

Comment:
Thank you for working on this! The text is truncated appropriately in this case, however, because `MultiSelect` is a component used in other areas of Studio, setting `max-width: 200px` seems to also affect the `/settings/#/storage` page where this component is also applied:

<img width="721" alt="Screenshot 2023-07-07 at 11 52 02 AM" src="https://github.com/learningequality/studio/assets/46411498/c27ffeb4-ef98-4cf2-b12a-0df4c8f0d37b">

Reply:
@Jaspreet-singh-1032 Good idea, that sounds like it should work!